src/client/game/nav.ts

```diff
@@ -140,8 +140,9 @@ export function showJoinBtn(team: Team) {
 }
 
 function showBtn(btn: HTMLButtonElement) {
-  btn.classList.remove("invisible");
-  btn.disabled = false;
+  const ele = btn;
```

Comment:
This is to fix eslint's "Assignment to property of function parameter" error, as it's considered bad practice.